py/sight/sight.py

```diff
@@ -645,16 +650,15 @@ def exit_block(self, label: str, obj: sight_pb2.Object, frame=None) -> None:
       obj.block_end.location_of_block_start = self.open_block_start_locations.get(
       )[-1]
 
-
       elapsed_time_ns = time.time_ns() - self.active_block_start_time.get()[-1]
```

Comment:
Should pull the elapsed time stuff into separate function. We can collect more metrics (e.e. memory use) and it should all be encapsulated